src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Requests/DataSetQueryLocation.cs

```diff
@@ -214,8 +213,7 @@ public record DataSetQueryLocationLocalAuthorityOldCode : DataSetQueryLocation
 {
     /// <summary>
     /// The old code (previously the LEA code) of the local authority.
-    /// This should be a 3 digit number (e.g. `318`) or be
-    /// a combination of two codes (e.g. `314 / 318`).
+    /// This is typically a 3-digit number (e.g. `318`).
```

Comment:
I wasn't too sure about this. I backpedalled on having this comment because in the future they may have further variations like combining three or more codes together. Whilst unlikely, it's not impossible so the current comment didn't feel right.

I've decided to change the wording up a bit more to: 

> This is typically a 3-digit number e.g. `318`, but may be a combination of multiple codes e.g. `318 / 319`.

Reply:
Cool sounds good!